x-pack/filebeat/input/cometd/input_test.go

```diff
@@ -363,9 +329,11 @@ func oauth2Handler(w http.ResponseWriter, r *http.Request) {
 		_, _ = w.Write([]byte(`[{"ext":{"replay":true,"payload.format":true},"minimumVersion":"1.0","clientId":"client_id","supportedConnectionTypes":["long-polling"],"channel":"/meta/handshake","version":"1.0","successful":true}]`))
 		return
 	case `{"channel": "/meta/connect", "connectionType": "long-polling", "clientId": "client_id"} `:
-		if called < 2 {
+		if called < uint64(inputType) {
```

Comment:
Can you rename `inputType` to something that better describes what it is? Maybe `expectedHTTPEventCount`?

Reply:
Sure.